### DISCUSSION TITLE
Toggle Switcher Widget

### DISCUSSION THREAD

**Discussion by idrissmatos** [elementor/elementor#22613] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Switching through different pricing plans, tabs, etc

### Describe the solution you'd like

Create a Widget switch: a monthly/yearly switch or on and off that will let visitors go through different pricing plans or add any other content using the flex box containers. https://webdesign.tutsplus.com/tutorials/pricing-table-with-monthly-yearly-css-toggle-switch--cms-93335

**Comment by 79ho3ein** [comment]
Hi @idrissmatos 

This widget was in Elementor Roadmap. But if I'm not mistaken, they stopped creating this widget. Because the structure of this widget is exactly the same as the new tab widget. Therefore, it does not seem logical to produce a new widget for it. Of course, this is my personal opinion.
The example website you sent above would be possible with Tabs + css widget. Of course, if the Elementor team adds [this feature](https://github.com/orgs/elementor/discussions/21627) for the tabs widget, it will be much easier.

Hi @nicholaszein👋, May I ask if my guess was correct? Has the team stopped releasing the content switcher widget?
